circuit_knitting/cutting/cutqc/__init__.py

```diff
@@ -37,9 +37,9 @@
 ]
 
 warn(
-    f"The package {__name__} is deprecated and will be removed no sooner than Circuit Knitting Toolbox 0.8.0. "
-    "There is a new circuit cutting implementation in circuit_knitting.cutting that implements similar and improved functionality, which will be maintained going forward."
-    " See https://qiskit-extensions.github.io/circuit-knitting-toolbox/circuit_cutting/tutorials/04_automatic_cut_finding.ipynb for a tutorial on automated cut-finding.",
+    f"The package {__name__} is deprecated and will be removed no sooner than Circuit Knitting Toolbox v0.8.0. "
+    "The circuit cutting workflow in circuit_knitting.cutting now implements similar and improved functionalities, which will be maintained going forward."
```

Comment:
IMO it's more clear with backticks around the module name
```suggestion
    "The circuit cutting workflow in `circuit_knitting.cutting` now implements similar and improved functionalities, which will be maintained going forward."
```